lbrynet/daemon/auth/client.py

```diff
@@ -130,7 +121,7 @@ def config(cls, key_name=None, key=None, pw_path=None,
             except AttributeError:
                 pass
             authpair = user + b':' + passwd
-            auth_header = b'Basic ' + base64.b64encode(authpair)
+            auth_header = b'Basic ' + base58.b58encode(authpair)
```

Comment:
In the issue comment you said that you'll be using b64 instead of b58, but is seems like opposite is done. Please crosscheck.

Reply:
I rebased this out (since it makes urls containing bad characters), will update the comment